backend/backend/models/auth.py

```diff
@@ -11,3 +11,4 @@ class User(Base):
     id = Column(Integer, primary_key=True)
     email = Column(String(length=255), unique=True, nullable=False)
     password_hash = Column(String, nullable=False)
+    name = Column(String(length=100), nullable=False)
```

Comment:
Names can be longer than 100 characters, so let's just remove that limit.
```suggestion
    name = Column(String, nullable=False)
```